assets/app/view/game/par_chart.rb

```diff
@@ -39,7 +39,7 @@ def par_choices
 
         cash = @current_entity.cash
         prices = @step.get_par_prices(@current_entity, @corporation_to_par)
-        prices.map! { |sp| "#{@game.format_currency(sp.price)} (#{(cash / sp.price.to_f).floor} shares)" }
+        prices.map! { |sp| "#{@game.format_currency(sp.price)} (#{[(cash / sp.price.to_f).floor, 10].min} shares)" }
         text = "#{@current_entity.name} can par #{@corporation_to_par.name} at #{prices.join(', ')}"
```

Comment:
this should probably be changed by code that returns the total number of shares a crop has. 
What's the easiest way to determine that?

Reply:
There isn't a great way. Maybe pass off of `@full_cap_event ? 10 : 5`

Reply:
Did you want to fix this or good with always saying 10? I don't have a strong preference.

Reply:
leaving as is. If another game with par view and has 5 share corps will want to change it, they can.